jellyfin.ruleset

```diff
@@ -15,6 +15,13 @@
     <Rule Id="SA1633" Action="None" />
   </Rules>
   <Rules AnalyzerId="Microsoft.CodeAnalysis.FxCopAnalyzers" RuleNamespace="Microsoft.Design">
+    <!-- disable warning CA1812: internal class that is apparently never instantiated.
+        If so, remove the code from the assembly.
+        If this class is intended to contain only static members, make it static -->
+    <Rule Id="CA1812" Action="Info" />
+    <!-- disable warning CA1822: Member does not access instance data and can be marked as static -->
```

Comment:
I get removing CA1812, but CA1822 might be worth keeping as a warning.

Reply:
We keep them as info, we plan on failing on warnings.